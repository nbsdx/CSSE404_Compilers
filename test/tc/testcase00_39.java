// bad return type

class A
{
   public static void main( String[] args )
   {
      int b = (new B()).start();
   }
}

class B
{
   public int start()
   {
      return true;
   }
}

