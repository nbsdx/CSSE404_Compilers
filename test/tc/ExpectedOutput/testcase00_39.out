Actual return type boolean of method start does not match declared type int
